data/index.json

```diff
@@ -177,11 +177,12 @@
     "name": "big-decent-clipped-38",
     "type": "table",
     "id": "big-decent-clipped-38",
+    "separator": ",",
     "description": "",
     "creator": "Alexander Lex",
     "path": "data_38.csv",
     "size": [
-      19,
+      102,
```

Comment:
I think the plan is to have the importer take care of this. If we're not using the importer though, we'll have to do it manually. 

The bizarre thing is that this file was created with the importer to my knowledge.